rome_app/views.py

```diff
@@ -598,12 +600,14 @@ def essay_detail(request, essay_slug):
     related_list=[]
     for work in essay.related_works():
         current_work={}
-        current_work['title']=work['primary_title']
-        current_work['creator']=work.get('creator')
+        current_work['title']=work['primary_title'].encode('ascii', 'ignore').decode('ascii')
```

Comment:
I was having an issue with stripping some of the encoding, but I just removed the encoding and decoding and it appears to be fine without that.